be/src/storage/persistent_index.cpp

```diff
@@ -1569,14 +1569,39 @@ Status PersistentIndex::load_from_tablet(Tablet* tablet) {
         // so we can load persistent index according to PersistentIndexMetaPB
         EditVersion version = index_meta.version();
         if (version == lastest_applied_version) {
-            status = load(index_meta);
-            LOG(INFO) << "load persistent index tablet:" << tablet->tablet_id() << " version:" << version.to_string()
-                      << " size: " << _size << " l0_size: " << (_l0 ? _l0->size() : 0)
-                      << " l0_capacity:" << (_l0 ? _l0->capacity() : 0)
-                      << " #shard: " << (_l1 ? _l1->_shards.size() : 0) << " l1_size:" << (_l1 ? _l1->_size : 0)
-                      << " memory: " << memory_usage() << " status: " << status.to_string()
-                      << " time:" << timer.elapsed_time() / 1000000 << "ms";
-            return status;
+            MutableIndexMetaPB l0_meta = index_meta.l0_meta();
+            if (l0_meta.format_version() != PERSISTENT_INDEX_VERSION_1) {
+                LOG(WARNING) << "different format version, we need to rebuild persistent index";
+                status = Status::InternalError("different format version");
+            } else {
+                status = load(index_meta);
+                LOG(INFO) << "load persistent index tablet:" << tablet->tablet_id()
```

Comment:
should only print this when status.ok()

Reply:
> should only print this when status.ok()

done, change inhttps://github.com/StarRocks/starrocks/pull/10244/commits/1753328b958be5c46afa3c3869c2617bd6450c99